### PR TITLE
TasitLinkWallet bootstrap

### LINKED ISSUE #265: `tasit-link-wallet` package bootstrap

### PR DESCRIPTION
**Pull request by marcelomorgado**
### Issue link

#265 

### Auto-close the issue?

Closes #265 

### Types of changes

New feature (non-breaking change that adds functionality)

Meta (a change that affects future changes to this repo)

Technical debt (a code change that doesn't fix a bug or add a feature but makes something clearer for devs)

### CODE REVIEW

**Comment by marcelomorgado** [comment]
A note: I created `TasitLinkWallet.test.js` because without that the test script fails. I'm not sure yet if we'll have that file as a test suite.

File: packages/tasit-link-wallet/README.md [hunk 1]
```diff
@@ -2,14 +2,6 @@
 
 This package helps with onboarding a user by linking a dapp with a wallet of their choosing. This can be done either using ERC20 and ERC721 approve functions or using WalletConnect.
 
-**Important note**
-
-This onboarding-related child package is still in a _coming soon_ state. If you're checking out this repo for the purposes of considering whether the code is up to your standards, here are some completed portions of the code base to check out:
```
Comment (pcowgill) [inline packages/tasit-link-wallet/README.md]: We might want to keep this here until there are some more substantive features in this package

Comment (marcelomorgado) [reply packages/tasit-link-wallet/README.md]: Done.

Comment (pcowgill) [reply packages/tasit-link-wallet/README.md]: Thanks. Resolving.

**Pull request merged**